### PR TITLE
chore(TheAnvil): Sync staging with internalstaging and deploy April release 2020.04

### PR DESCRIPTION
**Pull request by themarcelor**
Copying changes form internalstaging.

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# staging.theanvil.io/manifest.json
## Deployment changes
- guppy
  - Increased node.js max header size to prevent issue with large NIH login session cookies hitting request header limit. (uc-cdis/guppy#79)
- portal
  - Added footer link configuration to portal config: `components.footer.links` is an array of `{ text: <string, optional>, href: <string> }`. (uc-cdis/portal#661)
  - Added a new property to gitops.json: `components.login.contact_link: { text: string, href: string }`. If `login.contact_link` is present, it will be displayed instead of `login.email`. Email links can be represented in `contact_link` using "mailto:email@address" as the "href". (uc-cdis/portal#660)
  - Updated tiered access logic requires Guppy server >= 0.4.0 (uc-cdis/portal#650)
- sheepdog
  - The environment's user.yaml needs to be updated to add the new Sheepdog admin policy/resources/roles; for each user that has admin: true, grant them the new policy. See https://github.com/uc-cdis/commons-users/pull/810 for an example. (uc-cdis/sheepdog#319)

**Comment by PlanXCyborg** [comment]
# staging.theanvil.io/manifest.json
## Deployment changes
- guppy
  - Increased node.js max header size to prevent issue with large NIH login session cookies hitting request header limit. (uc-cdis/guppy#79)
- portal
  - Added footer link configuration to portal config: `components.footer.links` is an array of `{ text: <string, optional>, href: <string> }`. (uc-cdis/portal#661)
  - Added a new property to gitops.json: `components.login.contact_link: { text: string, href: string }`. If `login.contact_link` is present, it will be displayed instead of `login.email`. Email links can be represented in `contact_link` using "mailto:email@address" as the "href". (uc-cdis/portal#660)
  - Updated tiered access logic requires Guppy server >= 0.4.0 (uc-cdis/portal#650)
- sheepdog
  - The environment's user.yaml needs to be updated to add the new Sheepdog admin policy/resources/roles; for each user that has admin: true, grant them the new policy. See https://github.com/uc-cdis/commons-users/pull/810 for an example. (uc-cdis/sheepdog#319)

File: staging.theanvil.io/manifests/hatchery/hatchery.json [hunk 1]
```diff
@@ -0,0 +1,175 @@
+{
+  "user-namespace": "jupyter-pods-internalanvil",
+  "sub-dir": "/lw-workspace",
+  "user-volume-size": "500Gi",
+  "sidecar": {
+    "cpu-limit": "1.0",
+    "memory-limit": "512Mi",
+    "image": "quay.io/cdis/gen3fuse-sidecar:0.1.5",
```
Comment (emalinowski) [inline staging.theanvil.io/manifests/hatchery/hatchery.json:8]: Looks like this should be 2020.04

File: staging.theanvil.io/manifests/hatchery/hatchery.json [hunk 2]
```diff
@@ -0,0 +1,175 @@
+{
+  "user-namespace": "jupyter-pods-internalanvil",
```
Comment (emalinowski) [inline staging.theanvil.io/manifests/hatchery/hatchery.json:2]: This should be just jupyter-pods

File: staging.theanvil.io/manifests/hatchery/hatchery.json (hunk 1 above)
Comment (themarcelor) [reply staging.theanvil.io/manifests/hatchery/hatchery.json]: Good catch, pushing anther commit now.

**Comment by PlanXCyborg** [comment]
# staging.theanvil.io/manifest.json
## Deployment changes
- guppy
  - Increased node.js max header size to prevent issue with large NIH login session cookies hitting request header limit. (uc-cdis/guppy#79)
- portal
  - Added footer link configuration to portal config: `components.footer.links` is an array of `{ text: <string, optional>, href: <string> }`. (uc-cdis/portal#661)
  - Added a new property to gitops.json: `components.login.contact_link: { text: string, href: string }`. If `login.contact_link` is present, it will be displayed instead of `login.email`. Email links can be represented in `contact_link` using "mailto:email@address" as the "href". (uc-cdis/portal#660)
  - Updated tiered access logic requires Guppy server >= 0.4.0 (uc-cdis/portal#650)
- sheepdog
  - The environment's user.yaml needs to be updated to add the new Sheepdog admin policy/resources/roles; for each user that has admin: true, grant them the new policy. See https://github.com/uc-cdis/commons-users/pull/810 for an example. (uc-cdis/sheepdog#319)

**Pull request merged**